db/sql/sql.go

```diff
@@ -1343,7 +1343,7 @@ func (lk Like) ToSql() (sql string, args []interface{}, err error) {
 			return
 		}
 
-		expr = fmt.Sprintf("%s %s ? ESCAPE '\\'", key, opr)
+		expr = fmt.Sprintf("%s %s ? ESCAPE '\\\\'", key, opr)
```

Comment:
Tiny nit: you can use a raw string literal here to reduce the amount of backslashes e.g. ``` `%s %s ? ESCAPE '\\'` ```

Reply:
done